examples/rbd/storageclass.yaml

```diff
@@ -43,6 +43,13 @@ parameters:
    # By default it is disabled. Valid values are “true” or “false”.
    # A string is expected here, i.e. “true”, not true.
    # encrypted: "true"
+
+   # Use external key management system for encryption passphrases
+   # encryptionKMS: vault
+
+   # String representing KMS configuration. Should be unique and match ID in
+   # KMS ConfigMap.
```

Comment:
`Should be unique and match ID in KMS ConfigMap` what is ID here?

Reply:
this ID is just a correlation to the KMS config object in configMap (implemented this just by following pattern how ceph connection is configured)